mappings/net/minecraft/world/ModifiableWorld.mapping

```diff
@@ -12,6 +12,28 @@ CLASS net/minecraft/class_1945 net/minecraft/world/ModifiableWorld
 		ARG 2 drop
 		ARG 3 breakingEntity
 	METHOD method_8652 setBlockState (Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;I)Z
+		COMMENT Updates the blockstate at a position within this world calling all appropriate callbacks and event handlers.
+		COMMENT
+		COMMENT <p>When called on the server, the new blockstate is stored and propagated to clients and listeners as dictated
+		COMMENT by the supplied flags. Note that calling this on the client will update the world locally, but may not see the
+		COMMENT change persisted across loads. It's recommended to check whether running on the client or server before
+		COMMENT interacting with the world in this way.</p>
+		COMMENT
+		COMMENT <p>Accepted values of the flags are a bitset combination of the following:</p>
+		COMMENT <ul>
+		COMMENT <li>0b0000000 // 0 - DEFAULT/NONE - nothing is notified. The state is merely stored without any visual changes</li>
+		COMMENT <li>0b0000001 // 1 - PROPAGATE_CHANGE - Propagates a change event to surrounding blocks.</li>
```

Comment:
```suggestion
		COMMENT <li>1 << 0 // 1 - PROPAGATE_CHANGE - Propagates a change event to surrounding blocks.</li>
```
Same below, no need to write a stupid tons of zero. Just use leftshift to make things more clear!

Reply:
No. That makes it harder to interpret.